src/components/RemoveBtn.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const RemoveButton = ({ onClick, children }) => (
+  <button type="button" onClick={onClick}>
+    {children}
+  </button>
+);
```

Comment:
- [ ] [OPTIONAL] Instead of creating multiple Button components I'd suggest reusing one `<Button/>` component by passing in different props for adding and removing book since they essentially do the same thing

Reply:
Thanks.